sdk/PrebidMobileTests/Stubbing/ANHTTPStubURLProtocol.h

```diff
@@ -1,8 +1,4 @@
-<<<<<<< HEAD
-/*   Copyright 2014 Prebid.org, Inc.
-=======
 /*   Copyright 2017 APPNEXUS INC
```

Comment:
Use Prebid.org, Inc instead.

Reply:
And we should probably change all files to use Prebid.org, need to confirm with Matt.